recipe/meta.yaml

```diff
@@ -14,161 +14,157 @@
 {% endif %}
 
 package:
-  name: torchvision-split
+  name: torchvision
   version: {{ version }}
 
 source:
   url: https://github.com/pytorch/vision/archive/v{{ version }}.tar.gz
   sha256: d31fe52e4540750c8d372b0f38f1bfa81d8261193f2c2c06577332831d203c50
   patches:
+    # seems to be necessary for aarch???
     - 0001-Monkeypatch-TORCH_LIB_PATH.patch
+    # https://github.com/pytorch/vision/pull/8280
+    - 8280-remove_distutils.patch
+    # Test only patch below
+    # https://github.com/pytorch/vision/pull/8119
+    - 8119-use_integer_division_for_range_inputs.patch
+    - ignore_python312_deprecation_warnings.patch
 
 build:
-  number: 2
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  number: 3
   skip: true  # [win]
   # no CUDA-enabled pytorch on aarch yet
   skip: true  # [cuda_compiler_version != "None" and aarch64]
   # Upstream has specific compatability ranges for pytorch and python which are
   # updated every 0.x release. https://github.com/pytorch/vision#installation
   skip: true  # [py < 38]
-  skip: true  # [py > 311]
-  rpaths:
```

Comment:
Please leave the rpath bit

Reply:
Is it still important now that we have libtorch that installs things in the PREFIx/lib

Reply:
No. That's nice.

But it looks like pytorch needs a run-export on libtorch:
```
WARNING (torchvision,lib/python3.10/site-packages/torchvision/image.so): Needed DSO lib/libc10.so found in ['conda-forge/linux-64::libtorch==2.1.2=cuda112_hf51c5c5_301']
WARNING (torchvision,lib/python3.10/site-packages/torchvision/image.so): .. but ['conda-forge/linux-64::libtorch==2.1.2=cuda112_hf51c5c5_301'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
WARNING (torchvision,lib/python3.10/site-packages/torchvision/image.so): Needed DSO lib/libtorch_cpu.so found in ['conda-forge/linux-64::libtorch==2.1.2=cuda112_hf51c5c5_301']
WARNING (torchvision,lib/python3.10/site-packages/torchvision/image.so): .. but ['conda-forge/linux-64::libtorch==2.1.2=cuda112_hf51c5c5_301'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

Reply:
Or rather, we need to specify the host-dep on libtorch here.

Reply:
thanks!

Reply:
Do we still need the host-dep on pytorch? Can't we get rid of it thanks to the libtorch host-dep, and instead only have a run-dep on pytorch?

Reply:
> Do we still need the host-dep on pytorch? Can't we get rid of it thanks to the libtorch host-dep, and instead only have a run-dep on pytorch?

Yeah, I think we could, but it doesn't make a big difference for now, since we still need pytorch at runtime.